pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -106,11 +107,25 @@ type Container struct {
 	// TODO: SizeRw     *int64 `json:",omitempty"`
 	// TODO: SizeRootFs *int64 `json:",omitempty"`
 
-	// TODO: Mounts          []MountPoint
+	Mounts []MountPoint
 	// TODO: Config          *container.Config
 	NetworkSettings *NetworkSettings
 }
 
+// From https://github.com/moby/moby/blob/v20.10.1/api/types/types.go#L416-L427
+// MountPoint represents a mount point configuration inside the container.
+// This is used for reporting the mountpoints in use by a container.
+type MountPoint struct {
+	Type        string `json:",omitempty"`
+	Name        string `json:",omitempty"`
+	Source      string
+	Destination string
+	Driver      string `json:",omitempty"`
+	Mode        string
+	RW          bool
+	Propagation string
```

Comment:
Maybe you can drop Propagation field for quickly merging this PR ?

Reply:
I deleted the not compatibility properties, CI job cgroupv2 failed but I think it may not be related to this PR.